test/import/FSHImporter.Profile.test.ts

```diff
@@ -1352,7 +1352,7 @@ describe('FSHImporter', () => {
         assertFixedValueRule(profile.rules[0], 'identifier.system', 'http://example.org');
       });
 
-      it('should log an error and skip the rule when parsing fixed value Resource rule', () => {
+      it('should parsing fixed value Resource rule', () => {
```

Comment:
slightly awkward grammar here, maybe `should parse a fixed value Resource rule`?

Reply:
Oh yes wow I should have read that more carefully after deleting.